applications/nrf_desktop/doc/main.rst

```diff
@@ -0,0 +1,30 @@
+.. _nrf_desktop_main:
+
+Main module
+###########
+
+The main module is the entry point of the application.
+
+
+Module Events
+*************
+
++------------------+-------------+--------------------------+------------------------+------------------+
+| Source Module    | Input Event | This Module              | Output Event           | Sink Module      |
++==================+=============+==========================+========================+==================+
+|                  |             |  :ref:`nrf_desktop_main` | ``module_state_event`` |                  |
++------------------+-------------+--------------------------+------------------------+------------------+
+
+
+Configuration
+*************
+
+The module has no configuration options.
+
+
+Implementation details
+**********************
+
+
+The :ref:`nrf_desktop_main` module initializes the :ref:`event_manager` and then submits the module state event to inform the other modules that the main module is ready.
```

Comment:
My intention was to refer to type of the event but I know what you mean and can give name here.
As for module references - personally I would prefer to five references as it would be easier to navigate through documentation. We can fix it when we go and add prefix to all module names.